shell/platform/fuchsia/flutter/flatland_external_view_embedder.h

```diff
@@ -192,6 +192,9 @@ class FlatlandExternalViewEmbedder final
   SkISize frame_size_ = SkISize::Make(0, 0);
   float frame_dpr_ = 1.f;
 
+  // Set to true if input shield is enabled.
+  bool intercept_all_input_ = false;
```

Comment:
Could this be const? Then you don't need the safety default value, the compiler will enforce that a value is set in every ctor.

Reply:
Done